pkg/resources/schema.go

```diff
@@ -203,9 +204,22 @@ func ReadSchema(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	err = d.Set("data_retention_days", s.RetentionTime.Int64)
-	if err != nil {
-		return err
+	// "retention_time" may sometimes be empty string instead of an integer
+	{
```

Comment:
Is there a reason why you're using blocks nest and influence scoping here?

Reply:
There're multiple set and parse operations in this function. This scope is there to isolate the variables needed for retention time parsing logic from other set operations, just to be safe, not that there's shadowing.